packages/node/src/lib/process-response.ts

```diff
@@ -63,11 +63,11 @@ export default function processResponse(
     // This is the same thing that Node.js does internally:
     // https://github.com/nodejs/node/blob/9b8ba2536044ae08a1cd747a3aa52df7d1815e7e/lib/_http_server.js#L318
     statusText: res.statusMessage || STATUS_CODES[res.statusCode],
-    headers: objectToArray(headers),
+    headers: objectToArray(headers, true),
```

Comment:
Mind changing this API to be like this:

```suggestion
    headers: objectToArray(headers, { castToString: true }),
```

This is tough to skim over, and is an example of a [boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap). This is a nit though, I'm not _too_ bothered about it.